docs/src/simulation.md

```diff
@@ -0,0 +1,84 @@
+# ALBA Parameters Simulation
+
+ALBA provides strong security guarantees about the capability to generate a proof given some number of items \\(S_p\\):
+
+* a honest prover with more than \\(n_p\\) items has a probability of
+  failing to generate a proof lower than \\(2^{-128}\\),
+* symetrically, a (dishonest) prover with less than \\(n_f\\) items
+  has probability of succeeding to generate a proof lower than
+  \\(2^{-128}\\).
+
+This page provides a simple and user-friendly way of estimating how
+various ALBA parameters impact the size of the proof and probability
+of having a proof, given a number of items \\(S_p\\) varying between
+the two parameters' bounds. It is expected to be a quick way to
+estimate what would be "good" values for those parameters in a
+specific context, depending on security requirements and expected
+adversarial share.
+
+One can define:
+
+* \\(n_p\\): The size of the _honest set_ of items.
+* \\(n_f\\): The size of the _faulty set_ of items. \\(n_f\\) must be lower than \\(n_p\\).
+* Item size: The size of a single item, this is used to compute the full size of a proof.
+* \\(S_p\\): The number of items actually available to an adversary, or put differently, how much adversarial stake should we expect.
+
+From these parameters, we display a chart showing the probability of
+an adversary being able to compute a single valid proof, given some
```

Comment:
```suggestion
an adversary being able to compute a valid proof, given some
```
The probability term bounds the probability that an adversary can create *at least one* valid proof.